kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html

```diff
@@ -1,7 +1,7 @@
 <div class="image-info">
     <div class="image-info__group">
         <dl class="image-info__wrap metadata-line image-info__usage-rights">
-            <dt class="metadata-line__key">Usage rights category</dt>
+            <dt class="metadata-line__key">Rights & restrictions</dt>
```

Comment:
Interestingly, I just tried in the W3C validator and it's not (anymore?). `a & b` is seemingly valid. However `a &x b` is not.